pkg/proxy/metaproxier/meta_proxier.go

```diff
@@ -152,7 +152,7 @@ func (proxier *metaProxier) OnEndpointSliceAdd(endpointSlice *discovery.Endpoint
 	case discovery.AddressTypeIPv6:
 		proxier.ipv6Proxier.OnEndpointSliceAdd(endpointSlice)
 	default:
-		klog.V(4).Infof("EndpointSlice address type not supported by kube-proxy: %s", endpointSlice.AddressType)
+		klog.V(4).InfoS("EndpointSlice address type not supported", "type", endpointSlice.AddressType)
```

Comment:
Feel free to ignore it. Will the key "addressType" be a better fit here ?

Reply:
You are right! Have fixed it.